platform/features/plot/src/PlotOptionsForm.js

```diff
@@ -0,0 +1,166 @@
+/*****************************************************************************
+ * Open MCT Web, Copyright (c) 2014-2015, United States Government
+ * as represented by the Administrator of the National Aeronautics and Space
+ * Administration. All rights reserved.
+ *
+ * Open MCT Web is licensed under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ *
+ * Open MCT Web includes source code licensed under additional open source
+ * licenses. See the Open Source Licenses file (LICENSES.md) included with
+ * this source code distribution or the Licensing information page available
+ * at runtime from the About dialog for additional information.
+ *****************************************************************************/
+/*global define*/
+
+define(
+    [],
+    function () {
+        "use strict";
+
+        /**
+         * A class for encapsulating structure and behaviour of the plot
+         * options form
+         * @memberOf platform/features/plot
+         * @param topic
+         * @constructor
+         */
+        function PlotOptionsForm(topic) {
+            var self = this;
+            this.onchangeTopic = topic();
+
+            function onchange(value){
```

Comment:
Absolutely agree: should be able to do per-control change handling, at the same time, there should be a "bubble up" change handler for a given form so that it is not necessary to explicitly listen for changes on every field when you want "all fields in this form".